index.html

```diff
@@ -67,8 +80,8 @@ <h1>Multi-Post Stories</h1>
             </div>
         </div>
         <div class="grid-column">
-            <div class="item1">
-                <div class="item-block">
+            <div class="item1" >
```

Comment:
- [ ] Kindly ensure that when the project card is hovered on with the mouse the description and dark opacity disappears and a See Project button appears to the bottom of the image and takes the whole image width.
The mentioned button should take you to the Live Demo of the project or to the GitHub repository


.Your first project card on the row is not having the dark opacity as requred. On a clearer note, the first card on your website is what should appear after you hover the project cards


Before you hover 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/215233781-f76a0d21-776c-4e45-8fa7-fc4f201f8145.png)

After you hover 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/215233811-447ed0e6-8c79-40cf-9d33-29703ec3df23.png)